### ISSUE TITLE
Can't install PPSSPP on Android - "App not installed"

### ISSUE THREAD

**Issue by enomat** [hrydgard/ppsspp#13073]
Can't install PPSSPP on LG V20 Android 8. I get the `"App not installed"` message during installation.

The last good version:
**v1.9.3-904-g5367cdfdc**

The first bad version:
**v1.9.3-906-ge8b737226**

This means I can't install the new 1.10 version. I tried.

**Comment by unknownbrackets** [comment]
Normally this happens when switching between git and non-git versions.  You have to uninstall the old app first.  Does that help?

-[Unknown]

**Comment by enomat** [comment]
The problem was when I downloaded the apk to PC and transferred it through USB cable it somehow corrupted the files (I also tried other apps). Then I downloaded it throught Wifi and it installed. Then I tried another USB cable and it installed.

So the problem is that the **USB cable that came with my LG V20 H990N is faulty**. I also noticed weird connection problems with my PC using this cable. This cable has rounded shape. My other V20, the H990DS came with good cable, and it's flat shaped. If I use the H990DS flat shaped USB cable to transfer files from my PC the files do not get corrupted and can be installed.
